content/ai/ai-tools.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: AI Tools
+description: |
+  Access to large language models and AI tools through CCV's computing infrastructure.
+---
+
+<ContentSection title="About ChatCCV Tools">
+CCV provides access to cutting-edge AI tools and large language models through our high-performance computing
+infrastructure. Whether you need to run LLMs for research, explore AI capabilities, or integrate AI into your
+computational workflows, we have the resources and expertise to support your needs.
+
+<Button
+  variant="primary_outlined"
+  size="md"
+  href="https://docs.ccv.brown.edu/ai-tools/privacy-statement"
+>
+  CCV AI Tools Privacy Statement
+</Button>
+
+  <CardGroup>
+    <StyledCard
+      iconName="FaFileAudio"
+      title="Transcribe"
+      footer={<Button size="md" href="https://ai.ccv.brown.edu/transcribe">Try Transcribe</Button>}>
+
+      Transcribe audio files in over 20 different languages to text using state-of-the-art AI models such as Google
+      Gemini and OpenAI Whisper.
+    </StyledCard>
+  </CardGroup>
+</ContentSection>
+
+<ContentSection title="Experimental Tools">
```

Comment:
For this section, we should have a little blurb about experimenting with AI at CCV. And then instead of cards here, just have bullet points on LibreChat and Ask Oscar since we're not promoting those.